docs/content/Getting-Started/Cloud/02-Create-a-deployment.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Create a deployment
+permalink: /getting-started/cloud/create-a-deployment
+category: Getting Started
+subCategory: Cube Cloud
+menuOrder: 3
+isDisableFeedbackBlock: true
+---
+
+## Create deployment
+
+The first step in creating a deployment is to sign in to your Cube Cloud
+account. After signing in, click the **Create Deployment** button:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Landing Screen with no configured deployments"
+    src="https://ucarecdn.com/f3ec1ae0-1d5d-4b5b-9859-9fc145b38e43/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+Give the deployment a name and select the cloud provider and region of your
+choice and click **Next**:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Create Deployment Screen"
+    src="https://ucarecdn.com/2338323e-0db8-4224-8e7a-3b4daf9c60ec/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+## Set up Cube project
+
+Next, click **Create** to create a new project from scratch:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Upload Project Screen"
+    src="https://ucarecdn.com/46b72b61-b650-4271-808d-55203f1c8d8b/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+## Connect a data source
+
+The last step in creating a deployment is to connect your data source to Cube
+Cloud.
+
+<InfoBox>
+
+Want to use a sample database instead? Use the credentials below:
+
+<br />
+
+| Field    | Value              |
+| -------- | ------------------ |
+| Host     | `demo-db.cube.dev` |
+| Port     | `5432`             |
+| Database | `ecom`             |
+| Username | `cube`             |
+| Password | `12345`            |
+
+</InfoBox>
+
+To do this, first select the data source:
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Setup Database Screen"
+    src="https://ucarecdn.com/1d656ba9-dd83-4ff4-a59e-8b5f97a9ddcc/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+After selecting the data source, enter valid credentials for it and click
+**Apply**. Check the [Connecting to Databases][ref-conf-db] page for more
+details on specific data sources.
+
+<div style="text-align: center">
+  <img
+    alt="Cube Cloud Setup Database Screen"
+    src="https://ucarecdn.com/1ccdb149-bcf3-4426-9103-986f092939b7/"
+    style="border: none"
+    width="80%"
+  />
+</div>
+
+If you run into issues here, make sure to allow inbound connections from Cube
+Cloud IPs to access your database. This means you need to enable these IPs in
+your firewall. If you are using AWS, this would mean adding a security group
+with allowed IPs.
```

Comment:
It would be great to add a link to AWS docs on adding a security group with allowed IPs.